moderncvskillmatrix.sty

```diff
@@ -0,0 +1,1025 @@
+        \setlength{\cvskilllegend@rightdescriptorwidth}{\skilllegend@bodylength-\skilllegend@leftdesriptorfactor\skilllegend@bodylength-\cvskill@width-\skillmatrix@padding-2\skilllegend@padding}%
+    }%
+}%
+% 
+% %-------------------------------------------------------------------------------
+% %                \cvskilllegend 
+% %-------------------------------------------------------------------------------
+% default legend style
+% \cvskilllegend[*][<post_padding>][<first_level>][<second_level>][<third_level>][<fourth_level>][<fifth_level>]{<name>}
+\NewDocumentCommand\skillLegend@FontSize{}{\scriptsize}
+\DeclareDocumentCommand\cvskilllegend{s +O{} +O{} +O{} +O{} +O{} +O{} +m}{}%
+%   Print legend table explaining the meaning of \cvskill{1}...\cvskill{5}.
+% 
+%   Input:
+%       asterix (optional): include vertical (dashed) lines. This behaviour depends on the body style. 
+%                           For the fancy style, the asterix has no meaning.
+%       input_1 (optional): padding length appended to the legend, default: <0.25em>
+%       input_2 (optional): string describing meaning of \cvskill{1}, default: <basic knowledge>
+%       input_3 (optional): string describing meaning of \cvskill{2}, default: <intermediate knowledge with some project experience>
+%       input_4 (optional): string describing meaning of \cvskill{3}, default: <extensive project experience>
+%       input_5 (optional): string describing meaning of \cvskill{4}, default: <deepened expert knowledge>
+%       input_6 (optional): string describing meaning of \cvskill{5}, default: <expert/guru>
+%       input_7:            string giving a name to the legend, default: <>
+% 
+%   Example usages:
+%       \cvskilllegend{}    print default legend
+%       \cvskilllegend[0.5em]{Legend}   print default legend, but adjust post padding and display the word ``Legend''.
+%       \cvskilllegend*[0.2em][Grundkenntnisse][Grundkenntnisse und eigene Erfahrung in Projekten][Umfangreiche Erfahrung in Projekten][Vertiefte Expertenkenntnisse][Experte/Guru]{Legende}     Fully adjust legend to German example and including dashed lines
+% 
+%   Note:
+%       -The width of the columns can be adjusted by the \setcvskilllegendcolumns, see \setcvskilllegendcolumns
+%       -The design of the table is such that the <second_level> entry stands in the first column but 
+%       occupies rows two and three. This has been done to accomodate english and german defaults.
+%       An alternative is provided by \cvskillplainlegend
+
+% declare default legend entries
+\NewDocumentCommand\skillLegend@defaultLevelOne{}{basic knowledge}
+\NewDocumentCommand\skillLegend@defaultLevelTwo{}{intermediate knowledge with some project experience}
+\NewDocumentCommand\skillLegend@defaultLevelThree{}{extensive project experience}
+\NewDocumentCommand\skillLegend@defaultLevelFour{}{deepened expert knowledge}
+\NewDocumentCommand\skillLegend@defaultLevelFive{}{expert/guru}
```

Comment:
Can we go with `expert / professional` or `expert / specialist`? Not really a fan of `guru`. (And add spaces, looks better IMHO).

Reply:
Sure. That was the formulation I stole from a form given to me by a recruiting firm XD, but yeah, I can change it.